charts/templates/deployment.yaml

```diff
@@ -37,11 +50,10 @@ spec:
                 key: password
           {{ end }}
           command: ["/bin/sh"]
-          # Capture exit code from createsuperuser as it is not idempotent.
+          #TODO(jake): Masking createsuperuser with return code 0, this needs to be made idempotent.
           args:
             - -c
-            - >- 
-                python /coral-credits/manage.py migrate --run-syncdb &&
+            - >-
                 python /coral-credits/manage.py createsuperuser --no-input || echo $?
```

Comment:
Why does this need `echo $?`? Don't we want to expose the returncode directly?

Reply:
So we should add a comment here, but this is intentional because after the first run successful run, this command will fail, and we just ignore that.

Reply:
Ah, thanks for clarifying.